docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+name: vita-min
+
+services:
+  db:
+    container_name: db
+    image: postgis/postgis:13-3.4-alpine
+    user: postgres
+    ports:
+      - 5432:5432
+    volumes:
+      - database:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+  jobs:
```

Comment:
No change necessary, but naming this `worker` would be consistent with our Procfile-based deployment elsewhere (in production on Aptible, and in development via foreman)